docs/upgrade-notes.asciidoc

```diff
@@ -50,6 +50,37 @@ For Elastic Security solution release information, refer to {security-guide}/rel
 
 
 [discrete]
+[[breaking-201550]]
+.[Alerting] Legacy deprecations (9.0.0)
+[%collapsible]
+====
+*Details* +
+`POST /api/alerts/alert/{id?}` has been replaced by `POST /api/alerting/rule/{id?}`
+`GET /api/alerts/alert/{id}` has been replaced by `GET /api/alerting/rule/{id}`
+`PUT /api/alerts/alert/{id}` has been replaced by `PUT /api/alerting/rule/rule/{id}`
+`DELETE: /api/alerts/alert/{id}` has been replaced by `DELETE /api/alerting/rule/{id}`
+`POST /api/alerts/alert/{id}/_disable` has been replaced by `POST /api/alerting/rule/{id}/_disable`
+`POST /api/alerts/alert/{id}/_enable` has been replaced by `POST /api/alerting/rule/{id}/_enable`
+`GET /api/alerts/_find` has been replaced by `GET /api/alerting/rules/_find`
+`GET /api/alerts/_health` has been replaced by `GET /api/alerting/rule/_health`
+`GET /api/alerts/list_alert_types` has been replaced by `GET /api/alerting/rule_types`
+`POST /api/alerts/alert/{alert_id}/alert_instance/{alert_instance_id}/_mute` has been replaced by `POST /api/alerting/rule/{rule_id}/alert/{alert_id}/_mute`
+`POST /api/alerts/alert/{alert_id}/alert_instance/{alert_instance_id}/_unmute` has been replaced by `POST /api/alerting/rule/{rule_id}/alert/{alert_id}/_unmute`
+`POST /api/alerts/alert/{id}/_mute_all` has been replaced by `POST /api/alerting/rule/{id}/_mute_all`
+`POST /api/alerts/alert/{id}/_unmute_all` has been replaced by `POST /api/alerting/rule/{id}/_unmute_all`
+`POST /api/alerts/alert/{id}/_update_api_key` has been replaced by `POST /api/alerting/rule/{id}/_update_api_key`
+`GET /api/alerts/{id}/_instance_summary` has been deprecated without replacement
+`GET /api/alerts/{id}/state` has been deprecated without replacement
```

Comment:
I found this [replacement route](https://github.com/elastic/kibana/blob/493bd47db502aa98059ff75c32eedbc31bcfa726/x-pack/plugins/alerting/server/routes/get_rule_state.ts#L40) but it's internal and therefore the deprecation note. Wdyt?